### PR TITLE
Database users

### PR DESCRIPTION
**Pull request by jonycfu**
merge request. Thanks!

### CODE REVIEW

File: .env_sample [hunk 1]
```diff
@@ -1,7 +1,6 @@
...
 Afterwards create the following environment variables:
 
-GMAIL_DOMAIN=YOUR EMAIL DOMAIN HERE
-GMAIL_USERNAME=YOUR EMAIL USERNAME HERE
-GMAIL_PASSWORD=YOUR EMAIL PASSWORD HERE
-SUCCESS_URL=YOUR SUCCESS URL HERE
\ No newline at end of file
+GMAIL_DOMAIN=gmail.com
+GMAIL_USERNAME=vala.app.backend@gmail.com
+GMAIL_PASSWORD=valaapp1234
```
Comment (h8rry) [inline .env_sample:6]: Are these real passwords? `.env-sample` is a template. You should never commit real passwords to git!!!

File: .gitignore [hunk 2]
```diff
@@ -18,3 +18,4 @@
 
 .DS_Store
 .env
+config/database.yml
```
Comment (h8rry) [inline .gitignore:21]: Why are you git ignoring `database.yml`?

File: app/assets/javascripts/users.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (h8rry) [inline app/assets/javascripts/users.js:1]: If you don't need this, you should delete it

File: app/assets/stylesheets/users.css [hunk 4]
```diff
@@ -0,0 +1,4 @@
+/*
```
Comment (h8rry) [inline app/assets/stylesheets/users.css:1]: If you don't need this, you should delete it

**Comment by h8rry** [comment]
@jonycfu please review

File: .env_sample (hunk 1 above)
Comment (jonycfu) [inline .env_sample:6]: Sorry, harry. My terrible mistake. They have been removed as of now.

File: .gitignore (hunk 2 above)
Comment (jonycfu) [inline .gitignore:21]: This is b/c my postgres on linux defaults to expect a username for some reason. Even after setting the trust authentications.

**Pull request closed without merging**